src/pathutil/vcs.go

```diff
@@ -7,38 +7,36 @@ package pathutil
 import (
 	"os"
 	"path/filepath"
+	"strings"
 )
 
-var vcsDirs = []string{".git", ".svn", ".hg"}
-
-// FindVCSRoot find package root path from arg path
-func FindVCSRoot(basedir string) string {
-	var foundVCSDir bool
-
-	findvcsDirWalkFunc := func(path string, info os.FileInfo, err error) error {
-		if err != nil || info == nil || info.IsDir() == false {
-			return nil
-		}
-
-		for _, d := range vcsDirs {
-			_, err := os.Stat(filepath.Join(path, d))
-			if err == nil {
-				foundVCSDir = true
-				break
-			}
-		}
-
-		return nil
+func FindVCSRoot(root string) string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function FindVCSRoot should have comment or be unexported